### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mathgeek**
image

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -89,7 +87,6 @@ def update():
         python_dependency(repo.name, PYTHON_VERSION)
 
     python_dependency('heltour', PYTHON_VERSION)
-    local("pip install -r {}".format(project_relative("requirements.txt")))
```
Comment (mathgeek) [inline fabfile.py:92]: I'm seeing two of these failures (see screenshot) - not sure if it's just this one or if one of the calls above it is also trying to install the old requirements.txt. Guess I could try standing up the vagrant again and see what it does

File: fabfile.py [hunk 2]
```diff
@@ -89,7 +87,6 @@ def update():
         python_dependency(repo.name, PYTHON_VERSION)
 
     python_dependency('heltour', PYTHON_VERSION)
```
Comment (mathgeek) [inline fabfile.py:89]: @lakinwecker Is it necessary to have this and the above "container" at the same time? Also not entirely sure what purpose these serve

File: fabfile.py (hunk 1 above)
Comment (mathgeek) [reply fabfile.py:92]: image

Looks good

File: fabfile.py (hunk 2 above)
Comment (lakinwecker) [reply fabfile.py:89]: @mathgeek These jam a `.pth` file into the virtual environment which allows python to find the modules appropriately.

Comment (lakinwecker) [reply fabfile.py:89]: And yes, they're required.

**Review by lakinwecker — APPROVED** [review]

**Pull request merged**